gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
```

Comment:
Can we stick with all?

Reply:
Hmm it auto changes to `bin` after i run `./gradlew wrapper`. Do you know if that's the correct command to run?